pipeline/photo_cal.py

```diff
@@ -278,6 +278,9 @@ def run(self, *args, **kwargs):
             # Make the ZeroPoint object
             ds.zp = ZeroPoint( sources=ds.sources, provenance=prov, zp=zpval, dzp=dzpval,
                                aper_cor_radii=sources.aper_rads, aper_cors=apercors )
+            
+            ds.zp._upstream_bitflag = 0
+            ds.zp._upstream_bitflag |= sources.bitflag
```

Comment:
Photometric calibration depends also on astrometric calibration... so you'd want to add the `wcs` bitflag here as well.